Bio/SubsMat/FreqTable.py

```diff
@@ -88,6 +99,7 @@ def read_count(f):
 
 
 def read_freq(f):
+    """Read a frequency data file f and load values to the Frequency Table"""
```

Comment:
Missing full stop,

```
$ flake8 Bio/
Bio/SubsMat/FreqTable.py:102:1: D400 First line should end with a period
```

Reply:
flake has been a bit odd, not picking the missing DocStrings at first, so I haven't used on this file; anyway full stop added.